pkg/config/tlscfg/options.go

```diff
@@ -40,7 +40,7 @@ type Options struct {
 	MaxVersion     string        `mapstructure:"max_version"`
 	SkipHostVerify bool          `mapstructure:"skip_host_verify"`
 	ReloadInterval time.Duration `mapstructure:"reload_interval"`
-	certWatcher    *certWatcher  `mapstructure:"-"`
+	CertWatcher    *certWatcher  `mapstructure:"-"`
```

Comment:
no, the the right change would be to remove the tag, not to change the visibility of the field.